cyberark/conjur
# UX Updates 05

Closes:
- [341](https://github.com/cyberark/conjur/issues/341)
- [340](https://github.com/cyberark/conjur/issues/340)
- [339](https://github.com/cyberark/conjur/issues/339)

#### What does this pull request do?
Implements R2 of marketing feedback

#### Where should the reviewer start?
homepage

#### How should this be manually tested?
- Confirm that references to "followers" are removed from Scalability feature
- Confirm hero text has a comma after "machine based identity authorization, and..";
- Confirm references to OpenShift or CloudFoundry from Integrations feature
- Confirm sidebar & footer links for Conjur Enterprise link [here](https://www.cyberark.com/products/privileged-account-security-solution/cyberark-conjur/)
- Confirm that google analytics script is removed from <head>

On /welcome/how-conjur-works
- Confirm Next Steps link is named: CyberArk Conjur Overview
- Confirm Next Steps link is added: CyberArk Conjur Data Sheet

#### Screenshots (if appropriate)
![screenshot 2017-09-01 14 26 33](https://user-images.githubusercontent.com/123787/29982917-9ac8d91a-8f21-11e7-995f-37bf75ef52d1.png)

#### Link to build in Jenkins (if appropriate)
https://jenkins.conjur.net/job/cyberark--conjur/job/ux-updates-05/

#### Questions:
> Does this have automated Cucumber tests?
no

> Can we make a blog post, video, or animated GIF out of this?
no

> Is this explained in documentation?
no

> Does the knowledge base need an update?
no
